spec/index.html

```diff
@@ -8765,9 +8769,9 @@ <h5>Grouping and Aggregation</h5>
 Let E := [], a list of pairs of the form (variable, expression)
 
 If Q contains GROUP BY exprlist
-   Let G := Group(exprlist, P)
-Else If Q contains an aggregate in SELECT, HAVING, ORDER BY
-   Let G := Group((1), P)
+   Let G := Group(exprlist, ToList(P))
+Else If Q contains an aggregate in <code>SELECT</code>, <code>HAVING</code>, <code>ORDER BY</code>
```

Comment:
Removing the `<code>` elements again from the algorithm as it makes the algorithm look weird. Better come up with a proper approach to using `<code>` tags when addressing issue #101 
```suggestion
Else If Q contains an aggregate in SELECT, HAVING, ORDER BY
```